tests/BrowscapTest/Data/DataCollectionTest.php

```diff
@@ -825,6 +825,34 @@ public function testAddSourceFileThrowsExceptionIfDevicesEntryIsNotAnArray()
     }
 
     /**
+     * checks if an exception is thrown if the devices entry has multiple items and there is no #DEVICE# token
+     *
+     * @expectedException \LogicException
```

Comment:
Just a nitpicking, but should use​ `expectException` method invocation instead of annotation (same with message) in future